readthedocs/projects/models.py

```diff
@@ -362,7 +362,11 @@ class Project(models.Model):
 
     skip = models.BooleanField(_("Skip (disable) building this project"), default=False)
 
+    # null=True can be removed in a later migration
+    # be careful if adding new queries on this, .filter(delisted=False) does not work
+    # but .exclude(delisted=True) does!
```

Comment:
I think this will work fine, except for any projects created during the deploy window. But we can just plan to remove the null in next deploy.